### DISCUSSION TITLE
BSc template and courses

### DISCUSSION THREAD

**Discussion by Martafct** [joaomlourenco/novathesis#390] [Q&A] [answered]
Hello! I have a problem because I'm doing the report for BSc, and I think that the template does not exist. I tried to put BSc in the docdegree (1.novathesis.tex), but it gives error. Then, in the file 9_nova_fct.tex there are no courses for Bachelors in the university. 

So, I have to choose master's template, but I need for it to say: "Projeto final de Licenciatura xx" and not ""Mestrado em xx". Also, I don't need to have a speciality, under ""Mestrado em xx". 

I hope you can help me. Thank you. 
<img width="1440" alt="Captura de ecrã 2022-11-29, às 19 00 18" src="https://user-images.githubusercontent.com/115926172/204624197-d32a8832-23e2-44a0-879e-dad463f9bcd6.png">
<img width="747" alt="Captura de ecrã 2022-11-29, às 19 01 25" src="https://user-images.githubusercontent.com/115926172/204624220-820b90ef-0e19-44cb-969f-f05560cd1edf.png">

**Comment by joaomlourenco** [comment]
HI,
Thanks for your message here.  I knew the BSc was missing… but no one ever complained!  That means no BSc users up to now!!   :D
Please give me a couple of days and you'll have something “just for you”.  ;)
Otherwise I'll help you with the right phrasing in the MSc cover.

**Comment by joaomlourenco** [comment, marked as answer]
@Martafct,
I "designed" a new cover (in brown/orange) for the “bsc”, and also added the necessary strings in PT e EN.
Just select `\ntsetup{docdegree=bsc}` and check if it fits your needs… :)

_The best would be to download the new version from github, but if you feel adventurous you may try to download (and replace) the new/modified files:_
```
NOVAthesisFiles/Schools/nova/fct/Images/nova-fct-bsc-cover-back.pdf
NOVAthesisFiles/Schools/nova/fct/Images/nova-fct-bsc-cover.pdf
NOVAthesisFiles/Schools/nova/fct/Images/nova-fct-bsc-spine.pdf
NOVAthesisFiles/Schools/nova/fct/nova-fct-defaults.ldf
NOVAthesisFiles/Strings/strings-en.ldf
NOVAthesisFiles/Strings/strings-pt.ldf
```

**Reply by joaomlourenco** [reply]
I added yet another variant of the covers.
```
NOVAthesisFiles/Schools/nova/fct/Images/nova-fct-red-cover-back.pdf
NOVAthesisFiles/Schools/nova/fct/Images/nova-fct-red-cover.pdf
NOVAthesisFiles/Schools/nova/fct/Images/nova-fct-red-spine.pdf
```
  You can select them by adding to `Config/5_packages.tex`
 ```latex
\thesiscover(bsc,1-1,image):={nova-fct-red-cover}
\thesiscover(bsc,N-2,image):={nova-fct-red-cover-back}
\thesiscover(bsc,spine,image):={nova-fct-red-spine}
```

**Reply by Martafct** [reply]
Hello! Thank you so much for your help. But supposedly I need to have the cover in **blue** :/. And also, it need to say "Projeto Final" and not "Licenciatura". I'm sorry, I thought I said it. If you can help me with that it would be awesome.

**Reply by joaomlourenco** [reply]
To use the "blue" (msc) cover and use "Projeto Final", add to `Config/5_packages.tex`
```latex
\degreenameprefix(bsc,pt):={Projeto Final da Licenciatura em}.  % or "Projeto Final de"
\thesiscover(bsc,1-1,image):={nova-fct-msc-cover}
\thesiscover(bsc,N-2,image):={nova-fct-msc-cover-back}
\thesiscover(bsc,spine,image):={nova-fct-msc-spine}
```
